---
output: 
  bookdown::pdf_document2:
    latex_engine: xelatex
    keep_tex: true
    number_sections: false
    fig_caption: true
    toc: false
    includes:
        before_body: 
        - header.sty
geometry: "left=3cm,right=3cm,top=2cm,bottom=2cm"
linespacing: 1.5
fontsize: 12pt
mainfont: Palatino
bibliography: ../refs.bib
csl: ../university-of-york-harvard.csl
link-citations: true
citecolor: teal
linkcolor: teal
urlcolor: teal
header-includes:
- \usepackage[labelsep=period]{caption}
- \usepackage[labelfont=bf]{caption}
- \usepackage[switch]{lineno}
- \usepackage{type1cm} % scalable fonts
- \usepackage{lettrine}
- \usepackage{booktabs}
- \usepackage{sectsty} \sectionfont{\centering}
- \usepackage{caption}
- \captionsetup[figure]{font=footnotesize}
- \captionsetup[table]{font=footnotesize}
- \captionsetup[table]{justification=centerlast}
- \captionsetup[figure]{justification=centerlast}
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

```{r load-pkgs}
library(tidyverse) # Load the tidyverse. Functions are used for tidying data and creating plots via ggplot2.

library(kableExtra) # Load kableExtra. Table generation.

library(wordcountaddin) # Load wordcountaddin. Counts words and characters in text in an R markdown document.
```

```{r source}
source("../02-analysis.R", local = knitr::knit_global()) 
source("../03-analysis.R", local = knitr::knit_global()) 
```

# Introduction

## Atherosclerosis

## Flow

## Flow During Development

## Developmental Proteins / mechanosensors

## Endothelial

## Pathway

Does Axin, Angp2, Thrombosin-2 change if Wnt is inhibited?

## Hypothesis

XAV-939 Wnt/Beta Catenin inhibitor, acts by inhibiting tankyrase

# Methods

## Orbital Shaker

HUVECs were cultured in complete growth medium containing M199, sodium bicarbonate, pen-strep, amphotericin B, Hi-FBS, endothelial cell growth supplement (ECGS), and herparin. When \~80% confluent, cells were incubated with 1ml of trypsin until cells thoroughly detached, and neutralised with 9ml of M199. Cells were spun for 5 minutes at 400g to discard the supernatent, then re-suspended in M199 media before tranfering to 10mm radius 6 well plates. Once confluent, 3ml [@Warboys2019] of 0.1% DMSO in M199 or 0.1% XAV939 in M199 were each added to half of the plates [@Zhu2017]. Cells were then subjected to flow using a orbital shaker at 210 rpm for 72 hours, with the exception of a static controls.

## mRNA Isolation and qPCR

Cells were isolated from the periphery and centre of the plates with cold PBS and centrifuged for 5 minutes at 400g to remove the supernatant. Total mRNA was extracted using the RNEasy Mini Kit (Qiagen) and the concentration was determined spectrophotometrically. cDNA synthesis was performed using the Verso cDNA Synthesis Kit (Thermo Scientific) with 5.5μl of 0.01067% mRNA. *ANGPT2*, *AXIN2*, *THSB1*, and *HPRT1* mRNA was quantified using StepOne qPCR (Thermo Scientific) with SYBR Green, using oligonucleotide qPCR primers from Ensembl [@howe2020] (Table \@ref(tab:primers)).

```{r primers}
knitr::kable(
    data.frame(
        Gene = c(
            "ANGPT2",
            "ANGPT2",
            "AXIN1",
            "AXIN1",
            "HPRT1",
            "HPRT1",
            "THSB1",
            "THSB1"
        ),
        Direction = c("L", "R", "L", "R", "L", "R", "L", "R"),
        Sequence = c(
            "CGGCTGTGATGATAGAAATAGGGA",
            "GTTCCAAGAGCTGAAGTTCAAGTC",
            "TGTCACTTACTTTTTCTGTGGGGA",
            "TGTCACTTACTTTTTCTGTGGGGA",
            "TTGGTCAGGCAGTATAATCC",
            "GGGCATATCCTACAACAAC",
            "AAAGATGGAGAATGCTGAGTTGGA",
            "GGTTCCAAAGACAAACCTCACATT"
        )
    ),
    booktabs = T,
    caption = "Oligonucleotide qPCR primers from Ensembl."
) %>%
    collapse_rows(columns = 1, latex_hline = "none") %>%
    kable_styling(full_width = T, latex_options = "hold_position", position = "left") %>% 
    column_spec(column = 3, width = "8cm")
```

## Statistical Analysis

Relative expression is expressed as log2 2^ΔΔCt^ fold change ± SEM. Normality was determined with Kolmogorov-Smirnov Tests. Multiple-comparison analysis was performed using Kruskal-Wallis Test followed by post-hoc Dunn's Test. Comparisons to the static control were performed using one-way ANOVA followed by post-hoc Tukey's HSD test. Statistical analyses were performed in R [@R].

# Results

HUVECs were treated with XAV939 and exposed to low (centre) and high (periphery) shear stress for 72 hours using an orbital shaker. Expression of *ANGPT2*, *AXIN2*, and *THSB1* were then quantified with qPCR.

For each gene, expression was significantly lower than the static control in all conditions. (*P* \< 0.0001).

*ANGPT2* expression is higher in cells exposed to high flow compared with low flow (*P* \< 0.05). XAV393 upregulated *ANGPT2* in cells exposed to low stress (*P* \< 0.05 versus control), however, downregulated *ANGPT2* in cells exposed to high stress (*P* \< 0.05 versus control) (Fig. \@ref(fig:plots)A).

Whereas in *AXIN2* and *THSB1*, expression did not significantly differ between the control cells exposed to low or high stress, and XAV393 downregulated cells exposed to both low and high stress (*P* \< 0.05 versus control) (Fig. \@ref(fig:plots)BC).

```{r plots, fig.width = 10, fig.height = 8, fig.cap = "(**ABC**) Cells were  treated with DMSO(-) or XAV939(+) and exposured to low or high shear stress. Levels of angiopoeitin-2 , axin-2, and thrombospondin-1 mRNA quantified by qPCR. Data is shown as fold change ± SEM, normalised to the HPRT control and relative to the static control. $^{**}$*P* < 0.01, and $^*$*P* < 0.05. (**DEF**) Under the same conditions, gene expression of cells exposed to low stress are compared to cells exposed to high stress. Data is shown as fold change ± SEM, normalised to the HPRT control and relative to the periphery control. $^{**}$*P* < 0.01, and $^*$*P* < 0.05."}

plots <- ggarrange(plot2, plot3, nrow = 2)
ggsave("plots.png", plots, width = 10, height = 8)
plots
```

```{=html}
<!-- 
Clear and well-structured description of findings 
Appropriate narrative of the experimental rationale and the major conclusions 
May include descriptions of problem solving and troubleshooting
Well-prepared and consistently presented tables and figures 
Numbered tables and figures with legends (captions).
Legends should provide enough information to allow the reader to understand the table/figure.
Get your figures and tables ready first
Use enough text to provide a clear narrative through those figures and tables
Do start with some basic results
Highlight the key findings, but don’t discuss them
Make sure statistics are reported in the correct format
Few references -->
```
# Discussion

## Atheroprotective gene expression

Limitations of orbital shaker = imporve method

## Future

primer specificity test

epigenetics

look at proliferation, apoptosis, senescence, inflammation = PERP, p53

look at vascular repair = wound scratch assay?

look at emt = slug/snail?

```{=html}
<!-- Do not simply recapitulate the results
Explore the wider meaning of the results
To what extent has your investigation addressed the aims provided in the introduction?
How have your findings advanced understanding of the problem under investigation?
How do your findings relate to previous studies in this area?
Based on your findings, what new investigations might be conducted?
Work in parallel with the introduction
Don’t just give a commentary of the results
How does your work compare with previous work?
Why might you have got different results?
How could you improve the project? (Don’t just say ‘collect more data’!) -->
```
<!-- In future look at vascular repair with scratch wound assay.  Flow chamber more control over force nut expensive-->

# Acknowledgements

```{=tex}
\begin{flushright}
(`r word_count("report.Rmd")` Words) remove headers !!!
\end{flushright}
```
# References

```{=html}
<!--
# References
\setlength{\parindent}{-0.2in}
\setlength{\leftskip}{0.2in}
\setlength{\parskip}{8pt}
\vspace*{-0.2in}
\noindent
-->
```
