---
output: 
  bookdown::pdf_document2:
    latex_engine: xelatex
    keep_tex: true
    number_sections: false
    fig_caption: true
    toc: false
    includes:
        before_body: 
        - header.sty
geometry: "left=3cm,right=3cm,top=2cm,bottom=2cm"
linespacing: 1.5
fontsize: 12pt
mainfont: Palatino
bibliography: refs.bib
csl: nature-cell-biology.csl
link-citations: true
citecolor: teal
linkcolor: teal
header-includes:
- \usepackage[labelsep=period]{caption}
- \usepackage[labelfont=bf]{caption}
- \usepackage[switch]{lineno}
- \usepackage{draftwatermark}
- \SetWatermarkLightness{0.9}
- \usepackage{type1cm} % scalable fonts
- \usepackage{lettrine}
- \usepackage{booktabs}
- \usepackage{sectsty} \sectionfont{\centering}
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

```{r load-pkgs}
library(tidyverse) # Load the tidyverse. Functions are used for tidying data and creating plots via ggplot2.

library(kableExtra) # Load kableExtra. Table generation.

library(wordcountaddin) # Load wordcountaddin. Counts words and characters in text in an R markdown document.
```

# Introduction

```{=plain}
<!-- 
Description of the background to the investigation 
An account of relevant literature
A clear statement of the major aims (or hypotheses) of the investigation
What is the question your project is asking and why it is a good question?
It should explain the basic nature of the problem and how your work seeks to extend scientific knowledge in the area. 
Be concise. Space is limited, so make sure that the literature you cover is directly relevant to your project. 
Start broad, then focus in.
End with a clear statement of aims (and/or hypotheses).-->
```
# Methods

```{=plain}
<!-- 
A clear and concise description of the experimental methods used in the investigation.
Sufficient detail should be provided to enable the experiments to be reproduced.
Data figures and analysis should not be included in this section.-->
```
## Orbital Shaker

HUVECs were cultured in flasks until \~80% confluent. Cells were then washed with warmed PBS and incubated with 1ml of trypsin until cells thoroughly detached. M199 media was added to the cells, before being transferred to a falcon tube and spun for 5 minutes at 400g. The supernatant was discarded, and cells were re-suspended in M199 media and transferred to 10mm radius 6 well plates. Once confluent, 3ml of 1% DMSO in M199 was added to one half of the plates, and 3ml of 1% Wnt inhibitor in M199 to the other half. Cells were subjected to flow using a orbital shaker at 210 rpm for 72 hours, with the exception of a static control.

## mRNA Isolation and qPCR

Media was removed from the plates and cells were washed with cold PBS. Cells were isolated from the periphery and centre of the plates using 350μl of PBS, before centrifuging for 5 minutes at 400g and removing the supernatant. Total mRNA was isolated using the RNEasy Mini Kit (Qiagen) and concentration was determined using a spectrophotometer. mRNA was reverse transcribed to cDNA using the Verso cDNA Synthesis Kit (Thermo Scientific). The oligonucleotide qPCR primers were obtained from Ensembl[@howe2020] (Table \@ref(tab:primers)).

```{r primers}
knitr::kable(
    data.frame(
        Gene = c(
            "ANGPT2",
            "ANGPT2",
            "AXIN",
            "AXIN",
            "HPRT1",
            "HPRT1",
            "KLF2",
            "KLF2",
            "THSB1",
            "THSB1"
        ),
        Direction = c("L", "R", "L", "R", "L", "R", "L", "R", "L", "R"),
        Sequence = c(
            "CGGCTGTGATGATAGAAATAGGGA",
            "GTTCCAAGAGCTGAAGTTCAAGTC",
            "TGTCACTTACTTTTTCTGTGGGGA",
            "TGTCACTTACTTTTTCTGTGGGGA",
            NA,
            NA,
            NA,
            NA,
            "AAAGATGGAGAATGCTGAGTTGGA",
            "GGTTCCAAAGACAAACCTCACATT"
        )
    ),
    booktabs = T,
    caption = "Oligonucleotide qPCR primers from Ensembl."
) %>%
    collapse_rows(columns = 1, latex_hline = "none") %>%
    kable_styling(full_width = T, latex_options = "hold_position", position = "left") %>% 
    column_spec(3, width = "8cm")
```

# Results

```{=html}
<!-- 
Clear and well-structured description of findings 
Appropriate narrative of the experimental rationale and the major conclusions 
May include descriptions of problem solving and troubleshooting
Well-prepared and consistently presented tables and figures 
Numbered tables and figures with legends (captions).
Legends should provide enough information to allow the reader to understand the table/figure.
Get your figures and tables ready first
Use enough text to provide a clear narrative through those figures and tables
Do start with some basic results
Highlight the key findings, but don’t discuss them
Make sure statistics are reported in the correct format
Few references -->
```
# Discussion

```{=html}
<!-- Do not simply recapitulate the results
Explore the wider meaning of the results
To what extent has your investigation addressed the aims provided in the introduction?
How have your findings advanced understanding of the problem under investigation?
How do your findings relate to previous studies in this area?
Based on your findings, what new investigations might be conducted?
Work in parallel with the introduction
Don’t just give a commentary of the results
How does your work compare with previous work?
Why might you have got different results?
How could you improve the project? (Don’t just say ‘collect more data’!) -->
```
<!-- In future look at vascular repair with scratch wound assay.  Flow chamber more control over force nut expensive-->

# Acknowledgements

```{=tex}
\begin{flushright}
(`r word_count("report.Rmd")` Words) remove headers !!!
\end{flushright}
```
# References

```{=html}
<!--
# References
\setlength{\parindent}{-0.2in}
\setlength{\leftskip}{0.2in}
\setlength{\parskip}{8pt}
\vspace*{-0.2in}
\noindent
-->
```
